### ISSUE TITLE
CSC : warning CS9057: The analyzer assembly 'C:\Users\<removed>\FastEnum.Generators.dll' references version '4.11.0.0' of the compiler, which is newer than the currently running version '4.8.0.0'.

### ISSUE THREAD

**Issue by fschwiet** [xin9le/FastEnum#46]
First of all, thanks for your work making this Nuget package available. I recently migrated from 1.8 to 2.0 and started to get a build warning. It is just a warning, things still work (though I don't use the new preprocessor functionality so I can't vouch for that). I'm not sure what the error means, I think you may have built with a reference to something in net9.0, which is still in preview. If that is the issue it begs the question what earlier version you can/should reference, I don't know personally but I'm using net8.0 the latest production release.

My csproj file:

```
	<PropertyGroup>
		<TargetFramework>net8.0</TargetFramework>
		<LangVersion>11.0</LangVersion>
	</PropertyGroup>
...
	<ItemGroup>
		<PackageReference Include="FastEnum" Version="2.0.0" />
...
		<PackageReference Include="Roslynator.Analyzers" Version="4.12.6">
			<PrivateAssets>all</PrivateAssets>
			<IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
		</PackageReference>
	</ItemGroup>
```

**Comment by xin9le** [comment]
We suspect the issue may be attributed to FastEnum's utilization of `Microsoft.CodeAnalysis.CSharp` version `4.11.0`. In the near future, we intend to make adjustments to employ the lowest compatible version possible. While we cannot guarantee this will fully resolve the matter, it represents the extent of actions we can feasibly undertake at present.

**Comment by xin9le** [comment]
@fschwiet 
We've released version [2.0.1](https://github.com/xin9le/FastEnum/releases/tag/v2.0.1). We would appreciate it if you could test to see if this resolves the issue.

**Comment by fschwiet** [comment]
The issue is resolved, thanks again.
